src/components/timeline/Selection.js

```diff
@@ -59,7 +59,7 @@ type State = {|
 class TimelineRulerAndSelection extends React.PureComponent<Props, State> {
   _handlers: ?{
     mouseMoveHandler: MouseHandler,
-    mouseUpHandler: MouseHandler,
```

Comment:
Semantically speaking, I feel like `mouseup` event makes more sense for the selection component. But since `click` event is solving our problem, it works for me. I tested more and it looks like it's working well.